### ISSUE TITLE
`jb --version` reports 'dev'

### ISSUE THREAD

**Issue by davidham** [jsonnet-bundler/jsonnet-bundler#117]
Just downloaded 0.4.0. Expected it to report `0.4.0`.

**Comment by metalmatze** [comment]
Did you download one of the [pre-compiled binaries](https://github.com/jsonnet-bundler/jsonnet-bundler/releases/tag/v0.4.0), or where do you see this? Can you be more specific? Thanks!

**Comment by davidham** [comment]
I think I just ran the `go get` command from the readme. I tried it again just now with these results:
```
davidham at surprise in ~
$ GO111MODULE="on" go get github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb
go: found github.com/jsonnet-bundler/jsonnet-bundler/cmd/jb in github.com/jsonnet-bundler/jsonnet-bundler v0.4.0

davidham at surprise in ~
$ which jb
/Users/davidham/go/bin/jb

davidham at surprise in ~
$ jb --version
dev

```
Very possible (likely even!) that there is a PBCAK on my end, but this is how I get the `dev` result.

**Comment by sh0rez** [comment]
Yes, `go get` compiles the binaries from source, thus you get "dev" binaries.

Either pass the proper ldflags (see `Makefile`) to set the version variable, or download pre-compiled assets :)
